apps/condo/pages/analytics/detail/report_by_tickets/index.tsx

```diff
@@ -98,6 +98,9 @@ const tabsCss = css`
   & .ant-tabs-tab.ant-tabs-tab-active {
     font-weight: bold;
   }
+  & .ant-tabs-nav {
+    margin: 0;
```

Comment:
please notice, that our design system uses 2 margins for tab now: 40px for pages and 16px for components....I do not recommend to change margins to 0....

Fix margin-top of child container instead

(I'll do our custom tabs in free time, but for now it's like this)